### PR TITLE
Fix-create hazard report endpoint

### PR DESCRIPTION
**Pull request by francesc-pablo**

### CODE REVIEW

File: backend/src/models/hazardreport.ts
Comment (sosualfred) [inline backend/src/models/hazardreport.ts:1]: A user should be associated with each hazard report entry

File: backend/.env
Comment (sosualfred) [inline backend/.env:1]: Do not commit .env files to the repo

File: backend/src/models/hazardreport.ts
Comment (francesc-pablo) [reply backend/src/models/hazardreport.ts:1]: okay, got it

File: backend/.env
Comment (TheeDee6) [reply backend/.env:1]: I apologize for the confusion. The change to the .gitignore file was an unintentional push on my part. I’m sorry about that. I hope it does not affect productivity of the application.

**Review by sosualfred — APPROVED** [review]

**Pull request merged**